### DISCUSSION TITLE
allow-forms permission not set error for workload item condition.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/Microsoft-Fabric-workload-development-sample#141] [Q&A]
![image](https://github.com/user-attachments/assets/e3647bac-8e7e-4ced-ac92-744e5e5ddf2f)

I am seeing this error from today. When I am trying to save something directly to mu backend from a workload item. Is this a bug or new security restriction that fabric is gonna impose on the iframe.

**Comment by anitamayorov** [comment]
Hi @vinayslumel , 

The iframe is sandboxed and the allowed attributes are 'allow-scripts' and 'allow-same-origin', you can see more details here: https://learn.microsoft.com/en-us/fabric/workload-development-kit/extensibility-front-end#:~:text=Note,MDN%20web%20docs. These attributes will remain on the iframe.

**Reply by praveenn77** [reply]
<img width="723" alt="image" src="https://github.com/user-attachments/assets/58460c3c-e357-443d-ac12-5307cca934aa">

so we are removing all other items!!!

**Reply by anitamayorov** [reply]
Hi @praveenn77, the sandbox attributes are predefined by the host and cannot be changed by the iframe. The screenshot in our documentation has been updated to reflect this. https://learn.microsoft.com/en-us/fabric/workload-development-kit/extensibility-front-end#:~:text=The%20Sample%20Workload%20UI,to%20MDN%20web%20docs

**Reply by WilcoHendriksen** [reply]
So basically this means we cannot have forms that use submit? Is there a reason for this?

this fails...
`
        <form onSubmit={() => console.log("hi")}>
          <button type="submit">submit</button>
        </form>
`

**Reply by yonadiv** [reply]
You can implement this functionality yourself, but yes as we don't currently "allow-forms", the form submit will not work
